src/pages/[platform]/build-a-backend/auth/concepts/multi-factor-authentication/index.mdx

```diff
@@ -751,132 +1360,165 @@ func updateMFAPreferences() async throws {
     let authCognitoPlugin = try Amplify.Auth.getPlugin(
         for: "awsCognitoAuthPlugin") as? AWSCognitoAuthPlugin
 
-    let smsMfaPreference: MFAPreference = .enabled
-    let totpMfaPreference: MFAPreference = .preferred
+    let emailMfaPreference: MFAPreference = .preferred
 
     try await authCognitoPlugin?.updateMFAPreference(
-        sms: smsMfaPreference,
-        totp: totpMfaPreference)
+        email: emailMfaPreference)
 }
 ```
 </InlineFilter>
 
-<InlineFilter filters={["angular", "javascript", "nextjs", "react", "vue", "android"]}>
-If multiple MFA methods are enabled for the user, the `signIn` API will return `CONTINUE_SIGN_IN_WITH_MFA_SELECTION` as the next step in the auth flow. During this scenario, the user should be prompted to select the MFA method they want to use to sign in and their preference should be passed to `confirmSignIn`.
+## Set up a user's preferred MFA method
+
```

Comment:
can we add a quick blurb between these headings as sort of an intro to the content?

Reply:
[Added here](https://github.com/aws-amplify/docs/pull/8023/commits/aee8c2f7b04dcb5b29faf418d512ca44e35cae7b)